bin/bundle

```diff
@@ -1,3 +1,114 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
-load Gem.bin_path('bundler', 'bundle')
+# frozen_string_literal: true
```

Comment:
This is the bundler's version of binstub which should have better compatibility than the version introduced with Rails 7. Either way, it should have no noticeable affect.